js/bit2c.js

```diff
@@ -558,6 +559,32 @@ module.exports = class bit2c extends Exchange {
         }, market);
     }
 
+    isFiat (code) {
+        return code === 'NIS';
+    }
+
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        if (this.isFiat (code)) {
+            throw new NotSupported (this.id + ' fetchDepositAddress() does not support fiat currencies');
+        }
+        const request = {
+            'Coin': currency['id'],
+        };
+        const response = await this.privatePostFundsAddCoinFundsRequest (this.extend (request, params));
+        //     {'address': '0xf14b94518d74aff2b1a6d3429471bcfcd3881d42', 'hasTx': False}
```

Comment:
Sorry to nitpick.. can you put in the following format?
```suggestion
        //
        //     {
        //          'address': '0xf14b94518d74aff2b1a6d3429471bcfcd3881d42',
        //          'hasTx': False
        //      }
        //
```

Apart from that looks great to me! Thanks for the contribution!